reflect-kotlin/src/main/java/dagger/dagger.kt

```diff
@@ -0,0 +1,8 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
+package dagger
+
+import kotlin.reflect.KClass
+
+inline fun <C : Any> KClass<C>.create(): C = Dagger.create(java)
+inline fun <B : Any> KClass<B>.builder(): B = Dagger.builder(java)
```

Comment:
this "pollutes" the global namespace, would there be a way to do `inline fun <@Component.Builder B : Any> KClass<B>`? (even if that needs dagger contributions)